src/README.md

```diff
@@ -8,14 +8,17 @@ Django backend that provides a FHIR API for accessing data from the NDH database
 
 
 ## Contributing to ndh-api
-#### Prerequesites
+### Prerequesites
 - [docker](https://www.docker.com/)
 - [colima](https://github.com/abiosoft/colima) (if using macOS)
 - a postgres database with the ndh schema
 
-#### Local dev
-0. TEMP: Disable ZScaler (until)
```

Comment:
oh I have to relearn this _every time_: https://help.zscaler.com/zia/adding-custom-certificate-application-specific-trust-store is a good starting point.